db/migrate/20160218001400_add_default_follow_up_layout_id.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultFollowUpLayoutId < ActiveRecord::Migration
+  def change
+    add_reference :liquid_layouts, :default_follow_up_layout
```

Comment:
how does this work? how does liquid_layout know what table default_follow_up_layout indexes into? also should we have index: true?